packages/commands/src/bitcoin/inscription/call.ts

```diff
@@ -46,6 +46,18 @@ const main = async (options: CallOptions) => {
     );
 
     const revertAddress = options.revertAddress || address;
+    const abortAddress = options.abortAddress;
+    const revertMessage =
+      options.revertMessage !== undefined
+        ? Buffer.from(options.revertMessage, "utf8")
+        : undefined;
+
+    const revertOptions = {
+      abortAddress,
+      callOnRevert: true,
```

Comment:
I was under the impression that callOnRevert should be false when using Gateway.call.

Reply:
I suppose we might as well just set it to false ourselves to make it clearer. It requires testing the CCTXs again, so I propose we do it in a separate PR to make sure nothing breaks, https://github.com/zeta-chain/toolkit/issues/422